__tests__/setup/setupTestFramework.js

```diff
@@ -1,27 +1,18 @@
 import { Builder } from 'selenium-webdriver';
-import { configureToMatchImageSnapshot } from 'jest-image-snapshot';
 import { createServer } from 'http';
 import { join } from 'path';
 import { promisify } from 'util';
 import getPort from 'get-port';
 import handler from 'serve-handler';
 
+import { browserName } from '../constants.json';
 import createPageObjects from './pageObjects/index';
 import marshal from './marshal';
 import retry from './retry';
 import setupTestEnvironment from './setupTestEnvironment';
 
-const BROWSER_NAME = process.env.WEBCHAT_TEST_ENV || 'chrome-docker';
```

Comment:
Do we really want to remove this?

Reply:
Yes, this is moved to `constants.json` now.